src/components/navbar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
- [ ] Good job implement the pagination! Now, let's add all this functionality on the `App.js` script in place of the navbar, this is more according to good practice and an important requirement for this project. Add the browser route and others inside App.js, to call it on navbar you could have on the same script or call it with `import Link from 'react-router-dom'`. 🐶